src/main/java/iudx/catalogue/server/apiserver/CrudApis.java

```diff
@@ -53,6 +54,8 @@ public final class CrudApis {
   private static final Logger LOGGER = LogManager.getLogger(CrudApis.class);
   private boolean hasAuditService = false;
   private String host;
+  private Api api;
```

Comment:
This instance of Api is not being used anywhere. Can be removed
```suggestion
  private Api api;
```